.github/actions/javascript/getDeployPullRequestList/getDeployPullRequestList.ts

```diff
@@ -17,25 +16,26 @@ async function run() {
             await GithubUtils.octokit.actions.listWorkflowRuns({
                 owner: github.context.repo.owner,
                 repo: github.context.repo.repo,
+                // eslint-disable-next-line @typescript-eslint/naming-convention
                 workflow_id: 'platformDeploy.yml',
                 status: 'completed',
                 event: isProductionDeploy ? 'release' : 'push',
             })
         ).data.workflow_runs;
 
-        const priorTag = _.first(completedDeploys).head_branch;
+        const priorTag = completedDeploys[0].head_branch;
         console.log(`Looking for PRs deployed to ${deployEnv} between ${priorTag} and ${inputTag}`);
-        const prList = await GitUtils.getPullRequestsMergedBetween(priorTag, inputTag);
-        console.log(`Found the pull request list: ${prList}`);
+        const prList = await GitUtils.getPullRequestsMergedBetween(priorTag ?? '', inputTag);
+        console.log(`Found the pull request list: `, prList);
```

Comment:
```suggestion
        console.log(`Found the pull request list: ${prList}`);
```

Why we changed this?

Reply:
`prList` is a array of numbers and when it was in a template string the TS was complaining

Reply:
Sounds good to me, then let's get rid of the template string formatting, no?

```suggestion
        console.log('Found the pull request list:', prList);
```